app/assets/stylesheets/passengers.scss

```diff
@@ -34,9 +46,44 @@ a {
     color: #fff;
     background-color: #000;
   }
+  .radioDiv {
+   width:200px;
+  }
+  .leftRadio {
+   float:left;
+  }
+  .rightRadio {
+   float:right;
+  }
 }
 
-table{
+table#key {
+  td.space {
+    padding-left: 339px;
+  }
+  td.yellow {
```

Comment:
I would suggest using the same class names in your key as the main table itself (`.expired_within_grace_period`, `.no_note`, etc.)  Also, since all your `table#key td`s have the same `font-size` and `padding`, you don't have to repeat that part. Make a less-specific rule:

```scss
table #key {
  td {
    font-size: 13px;
    padding: 5px;
  }
}
```

Generally, your classes should describe what kind of thing it is, not how they look.  Consider the following: what if ST asks you to make expiring-soon passengers salmon?  Then you're left with

```scss
table#key {
  td.yellow {
    background: LightSalmon; //which isn't yellow
  }
}
```